bundles/org.openhab.binding.openthermgateway/src/main/java/org/openhab/binding/openthermgateway/internal/DataItemGroup.java

```diff
@@ -41,37 +41,37 @@ private static Map<Integer, DataItem[]> createDataItemGroups() {
                 new FlagDataItem(Msg.READ, ByteType.HIGHBYTE, 4, "ch2_enable", CodeType.B),
                 new FlagDataItem(Msg.READ, ByteType.HIGHBYTE, 4, "ch2_enablerequested", CodeType.T),
                 new FlagDataItem(Msg.READ, ByteType.HIGHBYTE, 4, "ch2_enableoverride", CodeType.R),
-                new FlagDataItem(Msg.READ, ByteType.HIGHBYTE, 5, "0x00:5", CodeType.B),
-                new FlagDataItem(Msg.READ, ByteType.HIGHBYTE, 6, "0x00:6", CodeType.B),
-                new FlagDataItem(Msg.READ, ByteType.HIGHBYTE, 7, "0x00:7", CodeType.B),
+                new FlagDataItem(Msg.READ, ByteType.HIGHBYTE, 5, "reserved_5", CodeType.B),
```

Comment:
Suggest you use 'reserved_0_5' rather than 'reserved_5' in order to differentiate between  'x00:5' and (say) 'x02:5' etc. Same comment applies to all the remaining item names below.

Reply:
Thanks, waiting on response from @jlaur before making this change

Reply:
Removed these message ids alltogether